### PR TITLE
CDAP-3407: Get cdap version in upgrade tool programmatically

### PR DESCRIPTION
**Pull request by rohitsinha54**

### CODE REVIEW

File: cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java [hunk 1]
```diff
@@ -96,8 +96,8 @@
    * Set of Action available in this tool.
    */
   private enum Action {
-    UPGRADE("Upgrades CDAP to 3.2\n" +
-              "  The upgrade tool upgrades the following: \n" +
+    UPGRADE("Upgrades CDAP to " + ProjectInfo.getVersion().toString() +
```
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java]: You have to call getVersion() once more, to get the string. Or implement the toString in Version.java.

Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java]: Sorry, disregard my comment. I was looking at this Version class as reference:
https://github.com/caskdata/cdap/blob/develop/cdap-proto/src/main/java/co/cask/cdap/proto/Version.java

The Version class you're using here has a toString implemented.

**Comment by anwar6953** [comment]
Why close?

**Comment by rohitsinha54** [comment]
Verifying messages on cluster

File: cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java [hunk 2]
```diff
@@ -260,6 +260,7 @@ private void doMain(String[] args) throws Exception {
             try {
               startUp();
               performUpgrade();
+              System.out.println("\nUpgrade completed successfully.\n");
```
Comment (anwar6953) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java]: do we need an extra newline?

Comment (rohitsinha54) [inline cdap-master/src/main/java/co/cask/cdap/data/tools/UpgradeTool.java]: Ya looks cleaner :)

**Comment by bdmogal** [comment]
:+1:

**Pull request merged**